### ISSUE TITLE
Can't re-install the latest version of DSE2.0

### ISSUE THREAD

**Issue by roberto-arista** [LettError/designSpaceRoboFontExtension#74]
For RoboFont 4.5b (build 2312131731) I ran into the following error

```
Installing 'Designspace Editor 2' report:
  Traceback (most recent call last):
      File "main.py", line 13, in <module>
    ImportError: cannot import name 'extensionIdentifier' from 'designspaceEditor' (/Users/benedikt/Documents/dev/RF4/plugins/DesignspaceEditor2.roboFontExt/lib/designspaceEditor/__init__.py)
  
```

thanks Benedikt for posting the issue on Discord!

**Comment by roberto-arista** [comment]
quick bugfix: replace the extension file manually, it should work
it's a "reinstall" issue

**Comment by typemytype** [comment]
mm I see... it imports from `designspaceEditor` which exists but not the new attribute in the module...

**Comment by typemytype** [comment]
should be fixed, please test, thanks!!

**Comment by roberto-arista** [comment]
fixed with 02628dbe505ff78054b12a65fe0b9468ef40639f
thanks!
